src/utils/hooks/api/use-get-collateral-currencies.tsx

```diff
@@ -1,14 +1,13 @@
-import { CollateralCurrencyExt, getCorrespondingCollateralCurrencies } from '@interlay/interbtc-api';
+import { CollateralCurrencyExt, getCollateralCurrencies } from '@interlay/interbtc-api';
 import { useQuery, UseQueryResult } from 'react-query';
 
-const getCollateralCurrencies = async (): Promise<Array<CollateralCurrencyExt>> => {
-  const nativeCollateralCurrencies = getCorrespondingCollateralCurrencies(window.bridge.getGovernanceCurrency());
-  const foreignCurrencies = await window.bridge.assetRegistry.getCollateralForeignAssets();
-  return [...nativeCollateralCurrencies, ...foreignCurrencies];
+const getCurrencies = async (): Promise<Array<CollateralCurrencyExt>> => {
+  const collateralCurrencies = await getCollateralCurrencies(window.bridge.api, window.bridge.assetRegistry);
+  return [...collateralCurrencies];
```

Comment:
should be able to condense this to something like this (ie. no need to spread before returning)
```
const getCurrencies = async (): Promise<Array<CollateralCurrencyExt>> => 
  getCollateralCurrencies(window.bridge.api, window.bridge.assetRegistry);
```

Reply:
Good spot :)